gslib/tests/test_signurl.py

```diff
@@ -296,7 +300,10 @@ def testSignurlGet(self):
 
   def testSignurlGetWithJSONKey(self):
     """Tests _GenSignedUrl with a GET method and the test JSON private key."""
-    expected = sigs.TEST_SIGN_URL_GET_WITH_JSON_KEY
+    if PY2:
```

Comment:
This can be removed since we're using the same set of hashes now per earlier PR feedback.